design-editor/src/panel/property/attribute/attribute-element-styles.js

```diff
@@ -76,7 +77,11 @@ class AttributeStyles extends DressElement {
                 if (styles[i].parentSelector && element
                     && element.closest(styles[i]['parent-selector']).length == 0) {
                     continue;
-                }
+				}
+				if (styles[i].version
+					&& !(styles[i].version === StateManager.get('screen').profile)) {
```

Comment:
Please check: component-element.js:37, there is a function meant to be used to filter filter based on version.
Please consider moving this function to some common module and reuse it.